lib/xml_builder/formatters/squeezed.ex

```diff
@@ -0,0 +1,3 @@
+defmodule XmlBuilder.Formatters.Squeezed do
```

Comment:
Would prefer the terminology `XmlBuilder.Formatters.Indented` and `XmlBuilder.Formatters.None`